### PR TITLE
add .env-example

### LINKED ISSUE #2: Make .env-example file
For local developers. Might as well make it available.

### PR DESCRIPTION
**Pull request by berit**
closes #2

### CODE REVIEW

**Comment by berit** [comment]
@plocket I wasn't sure if this was what you wanted or how to test it

File: pushing_testing_files/.env-example [hunk 1]
```diff
@@ -0,0 +1,11 @@
+PLAYGROUND_EMAIL=example@example.com
+PLAYGROUND_PASSWORD=12345
+PLAYGROUND_ID=1
+
+BASE_URL=https://www.example.com
```
Comment (plocket) [inline pushing_testing_files/.env-example:5]: How about https://www.docassemble-server-url.com?

File: pushing_testing_files/.env-example [hunk 2]
```diff
@@ -0,0 +1,11 @@
...
+REPO_URL=https://github.com/example/example
```
Comment (plocket) [inline pushing_testing_files/.env-example:6]: How about https://github.com/user_name/repo_name?

File: pushing_testing_files/.env-example [hunk 3]
```diff
@@ -0,0 +1,11 @@
...
+
+BRANCH_PATH=heads/master
```
Comment (plocket) [inline pushing_testing_files/.env-example:8]: How about repo_name/main?

**Review by plocket — APPROVED** [review]
This is looking pretty darn good. You're spot on. Not blocking, but I think we could improve the names in `.env` to guide the developer a bit more.

**Comment by berit** [comment]
@plocket updated the defaults mostly to your requests. Also added some comments

**Review by plocket — APPROVED** [review]
Double checked on `BRANCH_PATH` syntax and the one in here is right. We should make an issue to change the convention for the name to `BRANCH`. It probably wouldn't need an explanation at that point.

**Comment by plocket** [comment]
Did not know that comments in `.env` are now a Thing. Will wonders never cease...

**Comment by plocket** [comment]
Need to remember to add this to CHANGELOG on next version bump or something. And maybe to documentation.

**Pull request merged**